client/src/templates/Challenges/classic/editor.css

```
@@ -178,17 +178,16 @@ textarea.inputarea {
   }
 }
 
-.lower-jaw-icon-bar {
-  display: flex;
-  gap: 10px;
-  flex-direction: row;
-}
-.lower-jaw-icon-bar > button {
-  line-height: 0;
+.lower-utility-bar {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 100px);
```

To answer the question on discord you need to use `minmax`, to make it take full width, you will need to manipulate the `5em` value tho, to get a good looking button, and remove alot of properties


```suggestion
  grid-template-columns: repeat(auto-fill, minmax(5em,1fr));
```